src/testRunner/unittests/tsserver/partialSemanticServer.ts

```diff
@@ -111,6 +111,46 @@ import { something } from "something";
             assert.isTrue(hasException);
         });
 
+        it("allows syntactic diagnostic commands", () => {

```

Comment:
If there isn't already such a test, it would be nice to confirm that something sensible happens for semantic and suggestion diagnostics.

Reply:
The [test above (`throws on unsupported commands`)](https://github.com/microsoft/TypeScript/blob/09af37b0bba5dfdd5f64de865fa8092067125e93/src/testRunner/unittests/tsserver/partialSemanticServer.ts#L82-L112) should have a `SemanticDiagnosticsSync` request.